### ISSUE TITLE
Can't Install Leeflets Properly

### ISSUE THREAD

**Issue by ynkno13** [Leeflets/leeflets#98]
I downloaded the master download of Leeflets, downloaded it and installed it the way it's supposed to be installed, point my browser to the folder and it won't go to "index.php" without me telling it to. Then I get to the page and it says that I need to add my FTP details. I do all that, get to the page, publish it, then go back to the root domain name, I get a 404 error. I go into my FTP server, look at the folder, nothing is in the root folder except the Leeflets folder, which works just fine editing-wise. Can anyone help with this? This is the link: http://ajg.anthonyguidetti.com

**Comment by jmilone** [comment]
I get a 404 even when I append your URL with index.php. Can you double check to be sure your subdomain ajg.anthonyguidetti.com is mapped to the directory in which you are placing the Leeflets files? With some hosting companies, there will be a directory created specifically for the subdomain by default. It's usually the same name as the subdomain.

**Comment by ynkno13** [comment]
Oh, sorry, I forgot to mention that the folder with the Leeflets files is called "leeflets-master". Does this matter, or should I rename it to "leeflets"?

**Comment by jmilone** [comment]
Ah, well if you have Leeflets in a folder called "leeflets-master" then it will load in that folder. For example, this URL works fine: http://ajg.anthonyguidetti.com/leeflets-master/

If you want your Leeflets site to load for ajg.anthonyguidetti.com, then you will need to move all the files currently in /leeflets-master/ to the root folder. So, lets say your Leeflets files are currently in /public_html/leeflets-master/. You just need to move all the files from /public_html/leeflets-master/ and drop them directly in /public_html/

So, when you go in your root folder, you should just see leeflets files, including index.php

**Comment by bradt** [comment]
When you hit the publish button in Leeflets, it will generate an index.html file and place it in your root folder. So people will see your published site at http://ajg.anthonyguidetti.com/. You probably don't want to move Leeflets itself down to the root folder as it will have nowhere to publish the index.html file.

**Comment by ynkno13** [comment]
Yeah, I've been pressing the publish button, set the permissions to 777, and even put in the FTP account information. None of this is generating an index.html file.

**Comment by keegoid** [comment]
"When you hit the publish button in Leeflets, it will generate an index.html file and place it in your root folder."
This information should be in the instructions. It was confusing for me to understand too.
